README.md

```diff
@@ -1,5 +1,5 @@
 # REDCap OnCore Client
-This REDCap external module provides integration with Forte Research's OnCore. It allows REDCap project builders to associate a REDCap project with an OnCore protocol and import enrollment data into REDCap. It also allows developers to read data from OnCore via the SIP interface or the SOAP API.
+This REDCap external module provides integration with Forte Research's OnCore. It allows REDCap project builders to associate a REDCap project with an OnCore protocol and import enrollment data into REDCap. It also allows developers to read data from OnCore via the SIP interface, the SOAP API and UF's OCR API.
```

Comment:
I may be splitting hairs but should this be worded differently to be consistent with the "SIP or API" language used later?

e.g.:

```suggestion
This REDCap external module provides integration with Forte Research's OnCore. It allows REDCap project builders to associate a REDCap project with an OnCore protocol and import enrollment data into REDCap. It also allows users to select protocols via either the SIP interface or UF's OCR API, and read protocol data from OnCore via the SOAP API.
```